centreon/packages/ui/src/Graph/Chart/Chart.tsx

```diff
@@ -98,7 +100,9 @@ const Chart = ({
   thresholds,
   thresholdUnit,
   limitLegend,
-  skipIntersectionObserver
+  skipIntersectionObserver,
+  fx,
```

Comment:
we avoid abbreviated names , but since the useMatrix .. was not a custom hook maybe juste delete the suffix use , or give an understandable name for the fx ..